src/auto_controller/include/auto_controller/auto_controller_core.h

```diff
@@ -0,0 +1,24 @@
+#include "controller/utils.h"
+#include "jsoncpp/json/json.h"
+#include "jsoncpp/json/writer.h"
+#include <fstream>
+
+using namespace controller;
+
+#pragma once
+
+namespace auto_controller {
```

Comment:
Let all this be in the controller namespace. AutoController is just a specialized Controller class, what's the reason for a new namespace and module?